compute/rn/release-information/release-notes-22-06-update1.adoc

```diff
@@ -0,0 +1,55 @@
+== 22.06 Update 1 Release Notes
+
+The following table provides the release details:
+
+[cols="1,4"]
+|===
+|Build
+|22.06.197
+
+|Codename
+|Kepler, 22.06 Update 1
+//Tentative date
+|Release date
+|Jul 27, 2022
+
+|Type
+|Maintenance release
+
+|SHA-256 digest
+|5aa618314e176d03e559e58d2eba50959365cdc145cba99f5d47d90737d233bf
+|===
+
+// Besides hosting the download on the Palo Alto Networks Customer Support Portal, we also support programmatic download (e.g., curl, wget) of the release directly from our CDN:
+//
+// LINK
+
+=== Improvements, Fixes, and Performance Enhancements
+//GithubIssue PCSUP-issue
+// GH#39791 #PCSUP-10025 On-prem
+* Fixed the broken pipe error when https://prisma.pan.dev/api/cloud/cwpp/images#operation/get-images-download[downloading a large image CSV]. The error was fixed by extending the HTTP client timeout value.
+// GH#40032 #PCSUP-10097 On-prem only
+* Fixed the welcome tour screen for new users.
+// GH#39668 #PCSUP-9482
+* Fixed an issue wherein the Defenders blocked application deployments on SELinux due to incorrect SELinux labeling on proxy `runc`. The issue was fixed by applying the original runc's SELinux label to the created `runc` proxy binary.
+// GH#39821 On-prem only
+* Fixed the validity period error of https://docs.paloaltonetworks.com/prisma/prisma-cloud/22-06/prisma-cloud-compute-edition-admin/configure/certificates[self-signed certificates]. The value is extended and should be a whole number greater than or equal to 1.
```

Comment:
@Pubs-MV maybe we should mention that the limit was 365 and now it is unlimited

Reply:
How about - "The limit of 365 has been waved off and the value can now be a whole number greater than or equal to 1"?

Reply:
@divyabhushan sounds good